tests/regression_tests/deplete/test.py

```diff
@@ -130,3 +131,44 @@ def test_full(run_in_tmpdir, problem, multiproc):
 
     # Check that no additional tallies are loaded from the files
     assert np.all(n_tallies == 0)
+
+# Checks openmc.Materials objects can be created from depletion results
+def test_depletion_results_to_material(run_in_tmpdir, problem):
+
```

Comment:
Gives us two spaces between functions per pep8 format spec
```suggestion


def test_depletion_results_to_material(run_in_tmpdir, problem):
    """Checks openmc.Materials objects can be created from depletion results"""
```